pkgs/tools/security/stoken/default.nix

```diff
@@ -20,9 +20,8 @@ stdenv.mkDerivation rec {
     autoconf
   '';
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config autoconf automake libtool ];
```

Comment:
you can add `strictDeps = true` to prevent unintentional regression.

Reply:
Still builds.